### PR TITLE
[entities] Migrate to new client LanguageServiceClient in 0.16.2

### PR DESCRIPTION
**Pull request by gogasca**
Migrate analyze entities and analyze entities to read from GCS
Modify test cases to read single URI

### CODE REVIEW

File: language/src/analyze_entities.php [hunk 1]
```diff
@@ -39,26 +39,41 @@
...
-    $entities = $annotation->entities();
-    foreach ($entities as $entity) {
-        printf('Name: %s' . PHP_EOL, $entity['name']);
-        printf('Type: %s' . PHP_EOL, $entity['type']);
-        printf('Salience: %s' . PHP_EOL, $entity['salience']);
-        if (array_key_exists('wikipedia_url', $entity['metadata'])) {
-            printf('Wikipedia URL: %s' . PHP_EOL, $entity['metadata']['wikipedia_url']);
+    $languageServiceClient = new LanguageServiceClient(['projectId' => $projectId]);
+    try {
+        $entity_types = [
```
Comment (beccasaurus) [inline language/src/analyze_entities.php]: 😭

You should be able to use `Enum_Class::name($entity->getType())` 😄

Comment (beccasaurus) [reply language/src/analyze_entities.php]: e.g. as seen here: https://github.com/googleapis/gapic-generator/pull/2482/files#diff-e0f22c3a62995f9e35bf5c1caa4b5f8b

Comment (gogasca) [reply language/src/analyze_entities.php]: Hi Rebbeca, I tried to see if I could implement the suggestion but the class Type does not have a name method not sure If I'm missing something obvious?, I look for alternatives using ReflectionClass but needs additional library: http://php.net/manual/en/reflectionclass.getconstants.php, which I prefer not to add additional dependency

Comment (beccasaurus) [reply language/src/analyze_entities.php]: D'ohhhhh... I just saw that this isn't available until an upcoming version of protobuf is released for PHP (protobuf 3.7) – Hopefully _"~early January"_

/fyi @michaelbausor

Comment (beccasaurus) [reply language/src/analyze_entities.php]: Bleech, these are so gross. Copy/pasting enums from the website isn't ideal. But these will be able to be updated soon!!!

👍 Carry On.

Comment (gogasca) [reply language/src/analyze_entities.php]: Sounds good, I'' keep an eye once is updated I will create a new PR. Thanks

**Pull request merged**